File: Practica 1. POO/Practica 1. POO/Coordenadas.vb
Public Class Coordenadas
    Public punto1 As Double
    Public punto2 As Double
    Public punto3 As Double
    Public punto4 As Double

    Public Property Coordenada1() As Double
        Get
            Return punto1
        End Get
        Set(value As Double)
            punto1 = value
        End Set
    End Property

    Public Property Coordenada2() As Double
        Get
            Return punto2
        End Get
        Set(value As Double)
            punto2 = value
        End Set
    End Property

    Public Property Coordenada3() As Double
        Get
            Return punto3
        End Get
        Set(value As Double)
            punto3 = value
        End Set
    End Property

    Public Property Coordenada4() As Double
        Get
            Return punto4
        End Get
        Set(value As Double)
            punto4 = value
        End Set
    End Property

    Public Sub Coordenadas(ByVal Coordenadax1 As Double, ByVal Coordenadax2 As Double, ByVal Coordenaday1 As Double, ByVal Coordenaday2 As Double)
        punto1 = Coordenadax1
        punto2 = Coordenadax2
        punto3 = Coordenaday1
        punto4 = Coordenaday2
    End Sub
End Class

File: Practica 1. POO programacion orientada a objetos/Practica 1. POO programacion orientada a objetos/Coordenadas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Practica_1.POO_programacion_orientada_a_objetos
{
    class Coordenadas
    {
        private double punto1X;
        private double punto2X;
        private double punto3Y;
        private double punto4Y; 

    public double Coordenada1
        {
            get =>
                punto1X;
            set =>
                punto1X = value;
        }

        public double Coordenada2
        {
            get =>
                punto2X;
            set =>
                punto2X = value;
        }

        public double Coordenada3
        {
            get =>
                punto3Y;
            set =>
                punto3Y = value;
        }

        public double Coordenada4
        {
            get =>
                punto4Y;
            set =>
                punto4Y = value;
        }

        public void ConjuntoP (double Coordenadax1, double Coordenadax2, double Coordenaday1, double Coordenaday2)
        {
            punto1X = Coordenadax1;
            punto2X = Coordenadax2;
            punto3Y = Coordenaday1;
            punto4Y = Coordenaday2;
        }       
    }
}
